boards/nucleo-f103rb/Makefile.features

```diff
@@ -6,9 +6,6 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
-# load the common Makefile.features for Nucleo boards
```

Comment:
This change is unrelated. The commit that introduced it should be removed.

Reply:
Done.